docusaurus/docs/engineering/knowledge-base/development-lifecycle.md

```diff
@@ -4,154 +4,230 @@ sidebar_position: 7
 
 # Development Lifecycle
 
-To look at/download BobaBoard's code, check out our guide to [Setting Up your Coding Environment](/docs/engineering/start-developing/before-you-start).
-You might also be interested in [Git Notes](/docs/engineering/legacy-pages/git-notes)
+If you're new to collaborating on a code project, you may be wondering what that process looks like. This page gives an overview of the development lifecycle, explains key terminology, and guides you through the step-by-step process of contributing to BobaBoard's codebase.
 
-:::tip This seems scarier than it is!
-If you're new to coding and github, the following process might seem scary and overwhelming. Don't worry: it will become second nature very quickly! If you want someone to hold your hand throughout the process, contact Ms. Boba.
-:::
+To slowly get you familiar with more technical vocabulary you might run into, this document also includes relevant information about git and GitHub. For more information on these subjects you can look at our [Git Notes](/docs/engineering/legacy-pages/git-notes.md) page.
```

Comment:
Another small trick: try to put the important information at the beginning of the sentence.

```
This document also includes relevant information about git and GitHub to slowly get you familiar with more technical vocabulary you might run into.
```